go.mod

```diff
@@ -3,7 +3,8 @@ module github.com/redhat-developer/gitops-operator
 go 1.18
 
 require (
-	github.com/argoproj-labs/argocd-operator v0.0.16-0.20220516154109-5954e7f63a96
+	github.com/argoproj-labs/argocd-operator v0.0.16-0.20220527172704-5fbe1a9be37b
+	github.com/argoproj/argo-cd/v2 v2.3.4
```

Comment:
Just realized it added a dependency for Argo CD .. Is it required ? Let me hold the PR and verify.

Reply:
Removed the dependency, transitive dependency is upgraded.